app/views/guides/_form.html.erb

```diff
@@ -77,3 +77,35 @@
     </fieldset>
   <% end %>
 </div>
+
+<div class='well comments'>
+  <fieldset class='meta'>
+    <legend>Comments</legend>
+    <% if guide.latest_edition.present? %>
+      <% guide.latest_edition.comments.includes(:user).each do |comment| %>
```

Comment:
I think it would be better to make this query in the controller and expose it as an instance variable @comments